cmd/podman/system/reset.go

```diff
@@ -57,7 +57,7 @@ WARNING! This will remove:
 Are you sure you want to continue? [y/N] `)
 		answer, err := reader.ReadString('\n')
 		if err != nil {
-			fmt.Println(errors.Wrapf(err, "error reading input"))
```

Comment:
That'll change from stdout to stderr.

I am okay with it but want to highlight for others who might think differently.

Reply:
I think this is correct, since it is reporting and error.